nvdaHelper/remote/winword.cpp

```diff
@@ -788,14 +788,20 @@ void winword_getTextInRange_helper(HWND hwnd, winword_getTextInRange_args* args)
 		int curDisabledFormatConfig=0;
 		//generated form field xml if in a form field
 		//Also automatically extends the range and chunkEndOffset to the end of the field
-		BOOL isFormField=generateFormFieldXML(pDispatchRange,paragraphRange,XMLStream,chunkEndOffset);
+		const bool isFormField = TRUE == generateFormFieldXML(pDispatchRange,paragraphRange,XMLStream,chunkEndOffset);

```

Comment:
I introduced const since we do not need to modify this later, and this will stop us from doing so accidentally. I have converted from the `BOOL` integer type to the C++ boolean type for the type safety and readability that comes with it.